tap_salesforce/salesforce/__init__.py

```diff
@@ -381,13 +384,24 @@ def _get_selected_properties(self, catalog_entry):
                                             self.select_fields_by_default)]
 
 
-    def get_start_date(self, state, catalog_entry):
+    def get_start_date(self, state, catalog_entry, without_lookback=True):
```

Comment:
The Tap is using `get_start_date` in `sync.py` for bookmarking so, at that time, we do not want our bookmark to go backward. Thus, here we are getting the date without the lookback window therefore we have added an argument `without_lookback`. But to avoid confusion we have renamed that argument to `with_lookback`.

In the DoD of the card [TDL-16115](https://jira.talendforge.org/browse/TDL-16115). There is mentioned we need to add a default lookback of 10 seconds. So, we kept the default value of 10 rather than 0.